addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -8,28 +8,30 @@
             <div class="row align-items-center justify-content-between">
                 <!-- Desktop Mode -->
                 <nav aria-label="breadcrumb" class="col-md-8 d-none d-md-flex">
-                    <ol class="breadcrumb bg-transparent mb-0 ps-0 py-0 overflow-hidden">
-                        <li class="breadcrumb-item">
-                            <a href="/slides">Courses</a>
+                    <ol class="breadcrumb flex-nowrap bg-transparent mb-0 ps-0 py-0 overflow-hidden">
+                        <li class="breadcrumb-item flex-shrink-0">
+                            <a href="/slides" title="Courses">Courses</a>
```

Comment:
I think this requires a translation export.

Reply:
Hello @awa-odoo  :slightly_smiling_face: 
It seems like the translations are already present in the pot file :slightly_smiling_face: 
https://github.com/odoo/odoo/blob/15.0/addons/website_slides/i18n/website_slides.pot#L1910 
https://github.com/odoo/odoo/blob/15.0/addons/website_slides/i18n/website_slides.pot#L5006
Cheers!